sim/paladin/avengers_shield.go

```diff
@@ -9,9 +9,10 @@ import (
 
 func (paladin *Paladin) registerAvengersShieldSpell() {
 	baseCost := paladin.BaseMana * 0.26
-
 	baseModifiers := Multiplicative{}
 	baseMultiplier := baseModifiers.Get()
+	numHits := int32(1)
+        glyphedSingleTargetAS := paladin.HasMajorGlyph(41101)
```

Comment:
You should use the constants that are generated from proto for glyphs instead of a magic number